x-pack/platform/plugins/shared/onechat/public/application/components/conversations/new_conversation_prompt.tsx

```diff
@@ -66,45 +65,203 @@ export const NewConversationPrompt: React.FC<{}> = () => {
+            defaultMessage: 'Read the docs',
+          })}
+        </EuiButton>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+};
+
+const cards = [
+  // Create agent
+  {
+    key: 'createAgent',
+    title: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.agentCreation.title"
+        defaultMessage="Create a new agent"
+      />
+    ),
+    description: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.agentCreation.description"
+        defaultMessage="Build a custom agent tuned to your data and workflows."
+      />
+    ),
+    iconType: 'plus',
+    path: appPaths.agents.new,
+  },
+  // Manage agents
+  {
+    key: 'manageAgents',
+    title: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.agentManagement.title"
+        defaultMessage="Manage agents"
+      />
+    ),
+    description: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.agentManagement.description"
+        defaultMessage="View, edit, and organize your existing agents."
+      />
+    ),
+    iconType: 'controls',
+    path: appPaths.agents.list,
+  },
+  // Manage tools
+  {
+    key: 'manageTools',
+    title: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.toolManagement.title"
+        defaultMessage="Manage tools"
+      />
+    ),
+    description: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.toolManagement.description"
+        defaultMessage="Add, remove, or edit the tools your agents can use."
+      />
+    ),
+    iconType: 'wrench',
+    path: appPaths.tools.list,
+  },
+  // Documentation
+  {
+    key: 'documentation',
+    title: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.documentation.title"
+        defaultMessage="Read the docs"
+      />
+    ),
+    description: (
+      <FormattedMessage
+        id="xpack.onechat.welcome.quickNavigation.documentation.description"
+        defaultMessage="Learn more about building agents and tools."
+      />
+    ),
+    iconType: 'documentation',
+    // TODO: Add documentation link
+    path: '',
+  },
+];
+
+const QuickNavigationCards: React.FC<{}> = () => {
+  const { createOnechatUrl } = useNavigation();
+  const { euiTheme } = useEuiTheme();
+  const titleStyles = css`
+    ${useEuiFontSize('s')}
+    font-weight: ${euiTheme.font.weight.bold};
+  `;
+  const descriptionStyles = css`
+    ${useEuiFontSize('xs')}
+    color: ${euiTheme.colors.textSubdued};
+  `;
+  const iconBackgroundStyles = css`
+    padding: ${euiTheme.size.s};
+    background-color: ${euiTheme.colors.lightestShade};
+    border-radius: ${euiTheme.border.radius.medium};
+  `;
+  return (
+    <EuiFlexGroup gutterSize="s">
+      {cards.map(({ key, title, description, iconType, path }) => {
+        return (
+          <EuiFlexItem key={key}>
+            <EuiCard
```

Comment:
for a11y

role=button
aria-label=... 

this is just div so readers might nor recognize this as clickable?

Reply:
also `data-test-subj` for full story analytics and synthetics would be good to have